From: Hristo Venev <hristo@venev.name>
Date: Wed, 22 Apr 2020 16:23:33 +0300
Subject: dts: bcm2711: remove unused memory range

Physical addresses in the 0x0-0xfc00_0000 range correspond to SDRAM. They
have no place here.

Signed-off-by: Hristo Venev <hristo@venev.name>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi.dtsi b/arch/arm/boot/dts/bcm2711-rpi.dtsi
index b1115d6b9c77..5d68a3f8d7f6 100644
--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -187,8 +187,7 @@
 &scb {
 	ranges = <0x0 0x7c000000  0x0 0xfc000000  0x03800000>,
 		 <0x0 0x40000000  0x0 0xff800000  0x00800000>,
-		 <0x6 0x00000000  0x6 0x00000000  0x40000000>,
-		 <0x0 0x00000000  0x0 0x00000000  0xfc000000>;
+		 <0x6 0x00000000  0x6 0x00000000  0x40000000>;
 	dma-ranges = <0x0 0x00000000  0x0 0x00000000  0xfc000000>,
 		     <0x1 0x00000000  0x1 0x00000000  0x80000000>,
 		     <0x1 0x80000000  0x1 0x80000000  0x80000000>;
